app/src/main/java/eu/kanade/presentation/components/DropdownMenu.kt

```diff
@@ -26,3 +31,28 @@ fun DropdownMenu(
         content = content,
     )
 }
+
+@Composable
+fun RadioIcon(
```

Comment:
I meant just extract the icon part but this work as well just the naming should be a bit different imo
```suggestion
fun RadioButton(
```